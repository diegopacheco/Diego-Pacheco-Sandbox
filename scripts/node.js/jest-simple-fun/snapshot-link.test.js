import React from 'react';
import Link from './Link.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
