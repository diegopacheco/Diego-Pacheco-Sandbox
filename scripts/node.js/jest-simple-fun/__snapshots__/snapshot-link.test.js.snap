// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="http://www.facebook.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
